.github/workflows/ci.yaml

```diff
@@ -61,6 +61,8 @@ jobs:
           sudo find /usr/local/include -type d | xargs sudo chmod a+rx
           rm -f $PROTOC_ZIP
           ls /usr/local/include/google/protobuf/
+          go install google.golang.org/protobuf/cmd/protoc-gen-go@latest
```

Comment:
Move them to the proto gen script?

Reply:
Didn't work, got a permission denied error.